include/xrpl/protocol/STIssue.h

```diff
@@ -76,59 +94,66 @@ class STIssue final : public STBase, CountedObject<STIssue>
     friend class detail::STVar;
 };
 
+template <AssetType A>
+STIssue::STIssue(SField const& name, A const& asset)
+    : STBase{name}, asset_{asset}
+{
+    if (holds<Issue>() && !isConsistent(asset_.get<Issue>()))
+        Throw<std::runtime_error>(
```

Comment:
can you include this in unit tests (probably next to similar test as for serialisation) ?